support/cas-server-support-oidc-core-api/src/main/java/org/apereo/cas/oidc/web/controllers/token/OidcRevocationEndpointController.java

```diff
@@ -24,52 +18,17 @@
  * @author Misagh Moayyed
  * @since 5.2.0
  */
-@Slf4j
-public class OidcRevocationEndpointController extends BaseOAuth20Controller {
-    public OidcRevocationEndpointController(final OAuth20ConfigurationContext oAuthConfigurationContext) {
-        super(oAuthConfigurationContext);
-    }
-
-    /**
-     * Handle request for revocation.
-     *
-     * @param request  the request
-     * @param response the response
-     * @return the jwk set
-     */
-    @PostMapping(value = '/' + OidcConstants.BASE_OIDC_URL + '/' + OidcConstants.REVOCATION_URL)
-    public ResponseEntity<String> handleRequestInternal(final HttpServletRequest request,
-                                                        final HttpServletResponse response) {
-        try {
-            val authExtractor = new BasicAuthExtractor();
-            val credentialsResult = authExtractor.extract(new JEEContext(request, response, getOAuthConfigurationContext().getSessionStore()));
-            if (credentialsResult.isEmpty()) {
-                throw new IllegalArgumentException("No credentials are provided to verify revocation of the token");
-            }
+public class OidcRevocationEndpointController extends OAuth20RevocationEndpointController {
 
-            val credentials = credentialsResult.get();
-            val registeredService = OAuth20Utils.getRegisteredOAuthServiceByClientId(
-                getOAuthConfigurationContext().getServicesManager(),
-                credentials.getUsername());
-            val service = getOAuthConfigurationContext().getWebApplicationServiceServiceFactory().createService(registeredService.getServiceId());
-
-            val audit = AuditableContext.builder()
-                .service(service)
-                .registeredService(registeredService)
-                .build();
-            val accessResult = getOAuthConfigurationContext().getRegisteredServiceAccessStrategyEnforcer().execute(audit);
```

Comment:
I dont see any of these changes in the OAuth20 variant?

Reply:
You're right I forgot the audit, will add it soon.

Reply:
Just added it for public/passwordless clients only because this RegisteredServiceAccessStrategyEnforcer audit is already added for confidential clients by the OAuth20ClientIdClientSecretAuthenticator.

I think we could add a RevocationTokenResponseAuditResourceResolver or something like that in a future evolution ?